### ISSUE TITLE
Operation Canceled (Steam Deck) 

### ISSUE THREAD

**Issue by Tigray9** [rockerbacon/modorganizer2-linux-installer#686]
I'm sure it's been talked about already.
[canceled.txt](https://github.com/user-attachments/files/15856160/canceled.txt)

I was able to install MO2 in my Steam Deck before for Fallout 3 and New Vegas, but removed it because I wasn't have much use for it.

Now, whenever I attempt to install it again, it keeps saying "Operation canceled. Please see terminal for details."

Not sure what I'm doing wrong. Reinstalled protontricks and using latest MO2 version.

Should I use an older version? Any advice is welcome.

**Comment by swjzielinski** [comment]
>ln: failed to create symbolic link '/home/deck/.local/share/Steam/steamapps/compatdata/22380/pfx/dosdevices/y:': Read-only file system
try changing that folder to have write permissions.

**Comment by Tigray9** [comment]
Sorry for the late reply.
I tracked the dosdevices folder but see no 'y' file or folder. Only c, e, and z.

Should I make a y folder?

**Comment by swjzielinski** [comment]
No, that should be the symlink that can't be created. Basically every letter there corresponds to a drive inside the Wine system. 
There's another error in the file I didn't see:
>mkdir: cannot create directory ‘/home/deck/.local/share/Steam/steamapps/compatdata/22380/pfx/dosdevices/c:/windows/temp/_d3dcompiler_43’: Read-only file system

It's saying the same thing, about a different file. So, something above it is blocking it. 
Try doing
>chmod -R 755 /home/deck/.local/share/Steam/steamapps/compatdata/22380/pfx/dosdevices

and see if that changes anything. If not, I'd probably delete the 22380 folder and let Steam regenerate it.
